ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_linalg.py

```diff
@@ -768,3 +768,89 @@ def test_paddle_bincount(
         weights=None,
         minlength=0,
     )
+
+
+@st.composite
+def _x_without_unique_value(draw, dim):
+    x = []
+    for i in range(dim):
+        xi = draw(st.one_of(st.floats(allow_nan=False, allow_infinity=False)))
+        if len(x) > 0 and xi == x[-1]:
+            xi = draw(
+                st.one_of(
+                    st.floats(max_value=-2, allow_infinity=False, allow_nan=False),
+                    st.floats(min_value=2, allow_infinity=False, allow_nan=False),
+                )
+            )
+        x.append(xi)
+
+    return x
+
+
+@st.composite
+def _dtype_and_value_without_row_all_same_value(draw):
```

Comment:
If there a special use case fir this and the above function? If so you could move it to the `testing_helpers.py`

Reply:
Btw I don't see this referenced anywhere in the test

Reply:
No, actually I used it to see if the std=0 persist if we don't have same values along the axis. Removed it as it will create problems with shrinking in hypothesis.